src/ListPool/ListPool.cs

```diff
@@ -50,7 +49,6 @@ public ListPool(IEnumerable<TSource> source)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public void Add(TSource item)
         {
-            if (!_bufferOwner.IsValid) _bufferOwner = new BufferOwner<TSource>(MinimumCapacity);
             if (_itemsCount >= _bufferOwner.Buffer.Length) _bufferOwner.GrowDoubleSize();
```

Comment:
I mentioned about it, need to think is it a good idea or not to move control of buffer to bufferowner